packages/swap/src/adapters/api/dexhunter/types.ts

```diff
@@ -0,0 +1,229 @@
+export const Dex = {
+  Minswap_v1: 'MINSWAP',
+  Minswap_v2: 'MINSWAPV2',
+  Wingriders_v1: 'WINGRIDER',
+  Wingriders_v2: 'WINGRIDERV2',
+  Vyfi_v1: 'VYFI',
+  Sundaeswap_v1: 'SUNDAESWAP',
+  Sundaeswap_v3: 'SUNDAESWAPV3',
+  Splash_v1: 'SPLASH',
+  // TODO: @jorbuedo check if these are correct, is returning MUESLISWAP for sure
+  Muesliswap_clp: 'MUESLISWAP',
```

Comment:
@jorbuedo while testing the API `MUESLISWAP` was returning, can u confirm it? dunno about the V2, I followed their standard, waiting for confirmation.

Reply:
I see `MUESLISWAP` returned for orders, but can't get any new swaps to return their pools, even trading Ada/MILK which should have good liquidity there.
In fact, the list seems to have changed a bit since last time I saw them, Axo Trade, SnekFun, Splash Spectrum... 
I don't know how to keep that up to date correctly, I don't see any call in their app to fetch the dexes.

Reply:
With the unsupported, should be dealt with.